README.md

```diff
@@ -43,6 +43,28 @@ The documentation contains information about how to setup and run the two planne
 
 ## Installation
 
+### Script Installation (Recommended)
+1. Go to the [PX4 Development Guide](https://dev.px4.io/v1.8.2/en/setup/dev_env_linux.html#gazebo-with-ros) and run the Gazebo with ROS installation script
+1. Download [install_avoidance.sh](https://github.com/PX4/avoidance/blob/master/install_avoidance.sh)
+1. Run the script
+``` bash
+source install_avoidance.sh
+```
+To run the local planner
```

Comment:
I assume we just want to run one of the planners? Might be better to make that clear like this

1. Run the desired planner:
   * Local planner:
     ```bash
     roslaunch local_planner local_planner_sitl_3cam.launch
     ```
   * Safe landing planner
     ```bash
     roslaunch safe_landing_planner safe_landing_planner.launch
     ```
   * Global planner
     ```bash
     roslaunch global_planner global_planner_depth-camera.launch
     ```